### ISSUE TITLE
xcb_io.c abort the processs

### ISSUE THREAD

**Issue by willsir14** [awesomeWM/awesome#3219]
```
[xcb] Unknown sequence number while processing queue
[xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
[xcb] Aborting, sorry about that.
main: xcb_io.c:269: poll_for_event: Assertion `!xcb_xlib_threads_sequence_lost' failed.
```

This happens only in awesome wm
I have checked on other wm (dwm, bspwm) it works fine

**Comment by actionless** [comment]
never happened to me

what are the pre-requisites? what happens?"

**Comment by psychon** [comment]
What's your AwesomeWM version?

That error message comes from Xlib and if I did not do anything wrong, then AwesomeWM does not use Xlib since commit 0e81479a3f905f4551c88e4ad480972f1a9cff7b / version v4.0.

**Comment by willsir14** [comment]
![image](https://user-images.githubusercontent.com/36624796/100853784-42878c00-34b0-11eb-9bab-c5e8e6030f62.png)
@psychon 

@actionless , 
I tried to use libgraph using this package. 
installed using [this pkgfile](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=libgraph) changes dependencies guile1.8 to guile2.0 as recommended by [this](https://www.geeksforgeeks.org/add-graphics-h-c-library-gcc-compiler-linux/)
also added this line in line 16 `export CFLAGS="${CFLAGS} -fcommon"`
[main.c](https://github.com/awesomeWM/awesome/files/5628218/main.txt)
compiled using: gcc main.c -o main -lgraph

Its not working in awesome only. A blank black window is generated but not generating any graphics in that window.

**Comment by psychon** [comment]
Wait... where are you getting this error message? Is it AwesomeWM that aborts or is it this lgraph example thingie?

Also, what is the relation between that blank window and this abort? I am a bit confused about what is going on...

**Comment by willsir14** [comment]
I think awesomeWM is the one that aborts. I tried the same code in DWM and BSPWM it works.
```
[xcb] Unknown sequence number while processing queue
[xcb] Most likely this is a **multi-threaded client and XInitThreads** has not been called
[xcb] Aborting, sorry about that.
```
@psychon 
Is this problem of xcb?

**Comment by psychon** [comment]
No, the error message comes from Xlib (which AwesomeWM does not use): https://gitlab.freedesktop.org/xorg/lib/libx11/-/blob/78027fdb7aa07621a3b7cdf37050bae39012f2fb/src/xcb_io.c#L725-726 and https://gitlab.freedesktop.org/xorg/lib/libx11/-/blob/78027fdb7aa07621a3b7cdf37050bae39012f2fb/src/xcb_io.c#L34
